src/registry.ts

```diff
@@ -14,6 +14,12 @@ const defaultSetEntryOptions = {
   ...defaultOptions("/skynet/registry"),
 };
 
+// Regex for JSON revision value without quotes.
+const regexRevisionNoQuotes = /"revision":\s?([0-9]+)/;
+
+// Regex for JSON revision value with quotes.
+const regexRevisionWithQuotes = /"revision":\s?"([0-9]+)"/;
```

Comment:
The `?` matches only one space.
```suggestion
// Regex for JSON revision value without quotes.
const regexRevisionNoQuotes = /"revision":\s+([0-9]+)/;

// Regex for JSON revision value with quotes.
const regexRevisionWithQuotes = /"revision":\s+"([0-9]+)"/;
```